### PR TITLE
Implemeted Static UI for displaying parsed csv and Implemented logic …

### PR DESCRIPTION
**Pull request by disha1202**
…to map parsed Csv and implemented order state(#22c8vf8)

### CODE REVIEW

File: src/views/OrderDetail.vue [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+.search {
+  grid-area: search;
+}
+
+.filters {
+  grid-area: filters;
+}
+.list-item{
```
Comment (dt2patel) [inline src/views/OrderDetail.vue:131]: this is ok for now but add a note that this code needs to be updated to use universal list item class

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/store/modules/order/OrderState.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export default interface OrderState {
+    order: {
```
Comment (adityasharma7) [inline src/store/modules/order/OrderState.ts:2]: I think we should have state like this
```
list: {
        items: [],
        original: 0
      },
```

File: src/store/modules/order/getters.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { GetterTree } from "vuex";
+import OrderState from "./OrderState";
+import RootState from "../../RootState";
+
+const getters: GetterTree<OrderState, RootState> = {
+  getCsv(state){
```
Comment (adityasharma7) [inline src/store/modules/order/getters.ts:6]:
```suggestion
  getOrdeItems(state){
```

File: src/store/modules/order/actions.ts [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import OrderState from './OrderState'
+import * as types from './mutation-types'
+
+const actions: ActionTree<OrderState, RootState> = {
+  uploadCsv({commit}, payload){
```
Comment (adityasharma7) [inline src/store/modules/order/actions.ts:7]:
```suggestion
  updateOrderList({commit}, payload){
```

File: src/store/modules/order/mutation-types.ts [hunk 5]
```diff
@@ -0,0 +1,2 @@
+export const SN_ORDER = 'order'
+export const UPLOAD_CSV = SN_ORDER + '/UPLOAD_CSV'
```
Comment (adityasharma7) [inline src/store/modules/order/mutation-types.ts:2]:
```suggestion
export const ORDER_LIST_UPDATED = SN_ORDER + '/LIST_UPDATED'
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/router/index.ts [hunk 6]
```diff
@@ -32,6 +33,12 @@ const routes: Array<RouteRecordRaw> = [
     component: PurchaseOrder,
     beforeEnter: authGuard
   },
+  {
+    path: '/purchase-order-detail',
+    name: 'Purchase Order Detail',
```
Comment (ymaheshwari1) [inline src/router/index.ts:38]:
```suggestion
    name: 'PurchaseOrderDetail',
```

File: src/store/actions.ts [hunk 7]
```diff
@@ -1,6 +1,8 @@
 import { ActionTree } from 'vuex'
 import RootState from './RootState'
 
-const actions: ActionTree<RootState, RootState> = {}
+const actions: ActionTree<RootState, RootState> = {
+    
+}
```
Comment (ymaheshwari1) [inline src/store/actions.ts:6]: Remove these unwanted changes.

File: src/store/modules/order/actions.ts [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+      commit(types.UPLOAD_CSV,  payload );
```
Comment (ymaheshwari1) [inline src/store/modules/order/actions.ts:8]:
```suggestion
      commit(types.UPLOAD_CSV, payload);
```

File: src/store/modules/product/mutations.ts [hunk 9]
```diff
@@ -6,6 +6,14 @@ const mutations: MutationTree <ProductState> = {
   [types.PRODUCT_SEARCH_UPDATED] (state, payload) {
     state.products.list = payload.products;
     state.products.total = payload.totalProductsCount;
-  }
+  },
+  [types.PRODUCT_ADD_TO_CACHED_MULTIPLE] (state, payload) {
+    // TODO
```
Comment (ymaheshwari1) [inline src/store/modules/product/mutations.ts:11]: I think we can remove these TODO.

File: src/views/OrderDetail.vue [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+            <ion-label>{{ $t("Order buffer") }}</ion-label>
+            <ion-input type="text" value="50" /> 
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("Catalog") }}</ion-label>
+            <ion-select value="Backorder">
+              <ion-select-option>Backorder</ion-select-option>
+            </ion-select>
+          </ion-item>
+          <ion-button expand="block" fill="outline">Apply</ion-button>
```
Comment (ymaheshwari1) [inline src/views/OrderDetail.vue:31]: Internationalize text.

File: src/views/OrderDetail.vue [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+<script lang="ts">
+import Image from '@/components/Image.vue';
+
+import { defineComponent } from 'vue';
+import { mapGetters, useStore } from "vuex";
+import { useRouter } from 'vue-router';
+import { IonPage, IonHeader, IonToolbar, IonBackButton, IonTitle, IonContent, IonSearchbar, IonItem, IonThumbnail, IonLabel, IonInput, IonChip, IonIcon, IonButton, IonCheckbox, IonSelect, IonSelectOption } from '@ionic/vue'
+import { ellipsisVerticalOutline, sendOutline } from 'ionicons/icons'
+export default defineComponent({
+    components: {
```
Comment (ymaheshwari1) [inline src/views/OrderDetail.vue:75]: Change indentation to 2 spaces.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Comment by adityasharma7** [comment]
Changes merged with https://github.com/hotwax/import/pull/8. Closing

**Pull request closed without merging**